tests/unit/test_utils.py

```diff
@@ -462,13 +462,20 @@ def test_non_zero(self):
 class TestEncoding(object):
     """Tests for pip._internal.utils.encoding"""
 
-    def test_auto_decode_utf16_le(self):
+    def test_auto_decode_utf_16_le(self):
         data = (
             b'\xff\xfeD\x00j\x00a\x00n\x00g\x00o\x00=\x00'
             b'=\x001\x00.\x004\x00.\x002\x00'
         )
         assert auto_decode(data) == "Django==1.4.2"
 
+    def test_auto_decode_utf_16_be(self):
+        data = (
+            b'\xfe\xff\x00D\x00j\x00a\x00n\x00g\x00o\x00='
+            b'\x00=\x001\x00.\x004\x00.\x002'
+        )
```

Comment:
It might be good IMO to add the line `assert data.startswith(codecs.BOM_UTF16_BE)` so readers can know easily that the bytes object being tested has the expected properties (or alternatively concatenate).